website/docs/feature-flag-tutorials/python/implementing-feature-flags.md

```diff
@@ -0,0 +1,346 @@
+
+The token should have access to the “development” environment, as shown in the platform screenshot below.
+
+
+![Image of the API token creation form](/img/python-tutorial-create-api-token-form.png)
+
+
+The API token you generated can be managed in the API Access view in your project settings. It will become handy in Step 4.
+
+
+## 4. Add Unleash to a Python app
+
+In this section, you will clone an open-source Python application called [Flask Surveys Containers app](https://github.com/pamelafox/flask-surveys-container-app), which we are using to model a service that provides routing, serves HTML pages, and performs actions against a database. This app uses Flask, SQLAlchemy, and a PostgreSQL database.
+
+Use this command to clone the repository via your Terminal:
+
+```
+git clone git@github.com:pamelafox/flask-surveys-container-app.git
+```
+
+Next, navigate into your repository directory and create a `.env` file.
+
+Copy this code snippet into your `.env` file:
+
+
+```py
+FLASK_DEBUG=True
+DBHOST=db
+DBNAME=postgres
+DBUSER=app_user
+DBPASS=app_password
+```
+
+
+Next, install the Python SDK. Open your repository in a code editor and navigate to `requirements.txt` inside the `src` folder. Reference the Python SDK for installation.
+
+
+```py
+UnleashClient==5.11.1
+```
+
+
+In `src/backend/__init__.py`, import UnleashClient:
+
+
+```py
+from UnleashClient import UnleashClient
+```
+
+
+In the same file, call the Unleash client for initialization when the app runs with this code snippet:
+
+
+```py
+client = UnleashClient(
+ 	url="http://host.docker.internal:4242/api",
+    app_name="flask-surveys-container-app",
+ 	custom_headers={'Authorization': '<API token>'})
+ 
+client.initialize_client()
+```
+
+The URL will point your app to your Unleash instance through your Docker container for server-side communication.
+
+Replace the `<API token>` string in the Authorization header with the API token we created on our Unleash instance. This will allow the app to communicate with the Unleash API to use the feature flag we created.
+
+You can check our [API token and client keys documentation](https://docs.getunleash.io/reference/api-tokens-and-client-keys) for more specifics and see additional use cases in our [Server-Side SDK with Python](https://docs.getunleash.io/reference/sdks/python) documentation.
+
+Next, go to your terminal and build the app using this command:
+
+
+```
+docker-compose up --build
+```
+
+
+Navigate to [localhost://50505](http://localhost://50505) and the Surveys list should eventually display:
+
+
+![Image of Surveys app loaded in browser](/img/python-tutorial-surveys-app-loaded.png)
+
+
+Create 1 or more new surveys so they’re populated in your database!
+
+## 5. Use a feature flag to roll out a delete method
+
+In a real-world use case for your feature flag, you can gradually roll out new features to a percentage of users by configuring the flag's strategy.
```

Comment:
This paragraph and the one below it don't follow smoothly for me. How does gradual rollout in prod segue into controlling our delete case?